maldoca/ole/header_test.cc

```diff
@@ -27,11 +27,17 @@ namespace {
 using maldoca::OLEHeader;
 using maldoca::SectorConstant;
 
+#ifndef MALDOCA_CHROME
 std::string TestFilename(absl::string_view filename) {
-  return maldoca::file::JoinPath(
-      maldoca::GetRunfilesDir(),
-      absl::StrCat("maldoca/ole/testdata/", filename));
+  return maldoca::file::JoinPath(maldoca::GetRunfilesDir(),
+                        absl::StrCat("maldoca/ole/testdata/", filename));
 }
+#else
+base::FilePath TestFilename(absl::string_view filename) {
+  return maldoca::file::JoinPath(base::FilePath(maldoca::GetRunfilesDir()),
+                        base::FilePath(absl::StrCat("maldoca/ole/testdata/", filename)));
+}
```

Comment:
This is gonna break on windows since "maldoca/ole/testdata/" is not wstring, nor is filename and nor is getrunfiledir.  I think to the extend possible we should add wrappers in file.h to handle these things so it's not so hard to bring it back into google3.